src/wallet/sqlite.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <wallet/sqlite.h>
+
+#include <util/memory.h>
+#include <util/strencodings.h>
+#include <util/translation.h>
+#include <wallet/db.h>
+
+#include <stdint.h>
+
+static const char* DATABASE_FILENAME = "wallet.dat";
+
+SQLiteDatabase::SQLiteDatabase(const fs::path& dir_path, const fs::path& file_path, bool mock) :
+    WalletDatabase(), m_mock(mock), m_dir_path(dir_path.string()), m_file_path(file_path.string())
```

Comment:
1a2eec03f0dc7731d5c199a97db148534ce174a2, `clang-format` suggests
```suggestion
SQLiteDatabase::SQLiteDatabase(const fs::path& dir_path, const fs::path& file_path, bool mock)
    : WalletDatabase(), m_mock(mock), m_dir_path(dir_path.string()), m_file_path(file_path.string())
```

Reply:
Done